library/classes/rulesets/PQRS/reports/PQRS_0008/Exclusion1.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * PQRS Measure 0008 -- Exclusion 
+ *
+ * Copyright (C) 2015 - 2017      Suncoast Connection
+  * 
+ * LICENSE: This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0
+ * See the Mozilla Public License for more details. 
+ * If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ * 
+ * @author  Art Eaton <art@suncoastconnection.com>
+ * @author  Bryan lee <bryan@suncoastconnection.com>
+ * @package LibreHealthEHR 
+ * @link    http://suncoastconnection.com
+ * @link    http://librehealth.io
+ *
+ * Please support this product by sharing your changes with the LibreHealth.io community.
+ */
+
+class PQRS_0008_Exclusion1 extends PQRSFilter
+{
+    public function getTitle() 
+    {
+        return "Exclusion";
+    }
+    
+    public function test( PQRSPatient $patient, $beginDate, $endDate )
+    {
+$query =
+"SELECT COUNT(b1.code) as count".  
+" FROM billing AS b1".
+" JOIN form_encounter AS fe ON (b1.encounter = fe.encounter)".
+" WHERE b1.pid = ? ".
+" AND fe.date BETWEEN '".$beginDate."' AND '".$endDate."' ".
+" AND b1.code ='G8451';";
+
+$result = sqlFetchArray(sqlStatementNoLog($query, array($patient->id)));
```

Comment:
One thought I had. A lot of these are the same with the exception of the code. At some point in the future could you make a function to call and pass the needed variables?